templates/sections/event-body-two-column.html.twig

```diff
@@ -0,0 +1,18 @@
+{% set right_class = 'flex-lg-4-of-12' %}
+{% set left_class = 'flex-lg-7-of-12' %}
+
+<div{{ attributes.addClass('jumpstart-ui--two-column', 'flex-container', 'centered-container', 'su-event-content', settings.extra_classes) }}>
```

Comment:
This isn't really a jumpstart-ui--two-column layout. Are you sure you want this css class here?

Reply:
No I don't. I'll remove it. Good catch. Thank you